### DISCUSSION TITLE
can bpftrace support users  call  help functions in our script.

### DISCUSSION THREAD

**Discussion by miaochang123** [bpftrace/bpftrace#2704] [Q&A]
builtin functions  often can not  meet my  need , for example  I want to measure the rdtsc cost of my function ,  do bpftrace have this plan to support call bpf help functions ?

**Comment by viktormalik** [comment]
There is an ongoing work to allow users define their own functions inside bpftrace scripts (see #2516 for details). Once this is implemented, we plan to consider adding support for calling functions from external BPF programs, if that's what you're asking.
